css/home.css

```diff
@@ -34,17 +234,640 @@ Feel free to change the min/max-width values to whatever you need. Use variables
+		font-size: var(--font46);
+		line-height: var(--line-height50);
+		color: var(--color-onyx);
+	}
+	/* targeting the h4 stylings */
+	.what-we-do h4 {
+		margin-bottom: 0;
+		font-size: var(--font36);
+		line-height: var(--line-height40);
+		color: var(--color-white);
+	}
+	/* targeting the p styling to change the color */
+	.what-we-do p {
+		color: var(--color-white);
+	}
+
+	/* entire section */
+	.what-we-do {
+		padding-top: var(--spacing80);
+		padding-bottom: var(--spacing80);
+	}
+
+	/* the div holding the 3 cards */
+	.card-container {
+		width: 90%;
+		margin: 0 auto;
+		display: flex;
+		flex-direction: row;
+		justify-content: space-around;
+		align-content: center;
+	}
+	/* targeting the size of the individual cards */
+	.card-container .home-card {
+		width: 30%;
+		min-height: 530px;
+	}
+
+	/* ================================
+Our Values Section
+===================================  */
+
+	/* adjusting the background image for the entire section */
+	.values {
+		background-image: url(../assets/images/pages/home/values.jpg);
+		background-size: cover;
+		background-position: center;
+		padding-bottom: var(--spacing185);
+		padding-top: var(--spacing100);
+	}
+
+	/* the div holding both the title and the blue rectangle */
+	.values-holder {
+		margin: 0 auto;
+		width: 80%;
+	}
+	/* the div holding the title and the description */
+	.values .text-container {
+		width: 40%;
+		margin: 0;
+		text-align: left;
+	}
+	/* targeting the header style */
+	.values h4 {
+		margin-bottom: 0;
+		font-size: var(--font36);
+		line-height: var(--line-height40);
+		color: var(--color-white);
+	}
+	/* targeting the paragraph color */
+	.values p {
+		color: var(--color-white);
+	}
+	/* stylings for the blue rectangle  */
+	.our-values {
+		display: flex;
+		flex-direction: row;
+		justify-content: center;
+		width: 100%;
+		margin: 0;
+		padding-bottom: 60px;
+	}
+	/* stylings for the 3 divs within the blue rectangle  */
+	.our-values div {
+		display: flex;
+		flex-direction: column;
+		justify-items: center;
+		width: 30%;
+		text-align: left;
+		padding: 15px;
+	}
+
+	/* ================================
+Events and Programs Section
+===================================  */
+
+	/* stylings for the background for the entire section */
+	.events-and-programs {
+		background: url(../assets/images/pages/home/bg-overlay-vector-02.svg),
+			var(--color-bluepurple);
+		background-size: cover;
+		padding-top: var(--spacing20);
+	}
+	/* stylings for the div holding all 3 placeholders */
+	.events-container {
+		margin: 0 auto;
+		display: flex;
+		flex-direction: row;
+		justify-content: center;
+		align-items: center;
+		gap: 15px;
+	}
+	/* placeholder holding the image and event description */
+	.placeholder {
+		display: flex;
+		flex-direction: column;
+		width: 30%;
+	}
+	/* overriding global styles */
+	.events-and-programs h3,
+	.events-and-programs p {
+		color: var(--color-white);
+	}
+
+	.events-and-programs h4 {
+		color: var(--color-white);
```

Comment:
The color needs to change to white on hover.

<img width="521" alt="Screen Shot 2024-01-22 at 11 57 26 AM" src="https://github.com/indigenousfriends/itlct-website-redesign/assets/704697/f6adee7d-fdd1-4929-b1d6-a14df9a1a361">

Reply:
added "a.image-link" with a var of white and resolved the issue